packages/swap/src/event-handlers/refundEgressScheduled.ts

```diff
@@ -1,7 +1,9 @@
-import { swappingRefundEgressScheduled } from '@chainflip/processor/160/swapping/refundEgressScheduled';
+import { swappingRefundEgressScheduled as schema160 } from '@chainflip/processor/160/swapping/refundEgressScheduled';
+import { swappingRefundEgressScheduled as schema180 } from '@chainflip/processor/180/swapping/refundEgressScheduled';
+import z from 'zod';
 import type { EventHandlerArgs } from '.';
 
-const eventArgs = swappingRefundEgressScheduled;
+const eventArgs = z.union([schema180, schema160]);
```

Comment:
think we need to handle the new event shape here. typescript wont complain because of the `egressFee.toString()`

Reply:
true